posts/2019-12-30-reducing-support-for-32-bit-apple-targets.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Reducing support for 32-bit Apple targets"
+author: Pietro Albini
+---
+
+The Rust team regrets to announce that Rust 1.41.0 (to be released on January
+30th, 2020) will be the last release with the current level of support for
+32-bit Apple targets. Starting from Rust 1.42.0, those targets will be demoted
+to Tier 3.
+
+The decision was made on [RFC 2837], and was accepted by the compiler and
+release teams. This post explains what the change means, why we did it, and how
+your project is affected.
+
+[RFC 2837]: https://github.com/rust-lang/rfcs/pull/2837
+
+# What’s a support tier?
+
+The Rust compiler can build code targeting [a lot of
+platforms][platform-support] (also called “targets”), but the team doesn't have
+the resources or manpower to provide the same level of support and testing for
+each of them.
+To make our commitments clear, we follow a tiered support policy (currently
+being formalized and revised in [RFC 2803]), explaining what we guarantee:
+
+- Tier 1 targets can be downloaded through rustup and are fully tested
+  during the project’s automated builds. A bug or a regression affecting one of
+  these targets is usually prioritized more than bugs only affecting platforms
+  in other tiers.
+
+- Tier 2 targets can also be downloaded through rustup, but our
+  automated builds don’t execute the test suite for them. While we guarantee a
+  compiler build will be available, we don’t ensure it will actually work
```

Comment:
nit: some tier 2 targets are std only and we do not guarantee a compiler build

Reply:
Clarified this.